src/client/pages/SignInSuccess.tsx

```diff
@@ -1,233 +1,99 @@
-  position: relative;
-
-  &::before {
-    display: block;
-    content: '';
+const continueButton = css`
+  ${until.tablet} {
     width: 100%;
-    height: 1px;
-    background: ${neutral[86]};
-  }
-
-  ${from.desktop} {
-    font-size: 20px;
-
-    &::after {
-      display: block;
-      content: '';
-      position: absolute;
-      top: 0;
-      right: 100%;
-      width: 80px;
-      height: 1px;
-      background: ${neutral[86]};
-    }
+    justify-content: center;
   }
 `;
 
-const envelopeImageWrapper = css`
-  ${gridItem(envelopeSpanDefinition)}
-  display: flex;
-  justify-content: center;
-
-  ${until.desktop} {
-    padding-top: 52px;
-    border-bottom: 1px solid ${neutral[86]};
-    overflow: hidden;
-  }
-
-  ${from.desktop} {
-    order: 2;
-  }
-`;
-
-const envelopeImage = css`
-  width: 220px;
-  margin-bottom: -26px;
-`;
-
-const button = css`
-  margin-top: ${space[4]}px;
-`;
-
 type SignInSuccessProps = {
   geolocation?: GeoLocation;
   consents: Consent[];
 };
 
-export const SignInSuccess = ({ consents }: SignInSuccessProps) => {
-  const { id, name, description, consented } = consents[0];
+export const SignInSuccess = ({
+  geolocation,
+  consents,
+}: SignInSuccessProps) => {
+  const autoRow = getAutoRow(1, gridItemColumnConsents);
   const clientState: ClientState = useContext(ClientStateContext);
   const { pageData = {}, queryParams } = clientState;
-  const { page = '', geolocation } = pageData;
-  const renderForm = () => (
-    <form
-      css={formStyles}
-      action={buildUrlWithQueryParams('/signin/success', {}, queryParams)}
-      method="post"
-      onSubmit={({ target: form }) => {
-        onboardingFormSubmitOphanTracking(
-          page,
-          pageData,
-          // have to explicitly type as HTMLFormElement as typescript can't infer type of the event.target
-          form as HTMLFormElement,
-        );
-      }}
-    >
-      <CsrfFormField />
-      <h2 css={consentTitle}>{name}</h2>
-      <p css={consentDescription}>{description}</p>
-      <Checkbox
-        name={id}
-        value={id}
-        label="Yes, sign me up"
-        defaultChecked={!!consented}
-      />
-      <Button type="submit" cssOverrides={button}>
-        Continue to The Guardian
-      </Button>
-    </form>
-  );
 
   return (
     <>
       <Header geolocation={geolocation} />
-      <main>
-        <div css={titleWrapper}>
-          <div css={[autoMargin, gridRow]}>
-            <h1 css={title}>Get the latest offers sent to your inbox</h1>
-          </div>
-        </div>
-        <div css={formBackground}>
-          <div css={[autoMargin, gridRowPadding]}>
-            <div css={formWrapper}>
-              <div css={[innerGridRow, formGridRow]}>
-                <div css={envelopeImageWrapper}>
-                  <EnvelopeImage invertColors cssOverrides={envelopeImage} />
-                </div>
-                {renderForm()}
-              </div>
+      <main css={mainStyles}>
+        <form
```

Comment:
I wonder if this can make use of our central `MainForm` component. I don't think it really needs changing, but thought I'd throw the idea in there

Reply:
Intending to clean up the shared styles / logic as part of [this card](https://trello.com/c/ncPxVzLu/3810-onboarding-flow-redesign-follow-up).